package.json

```diff
@@ -44,12 +44,8 @@
     "storybook": "storybook dev -p 6006"
   },
   "devDependencies": {
-    "@storybook/manager": "^8.0.0-rc.2",
-    "@storybook/manager-api": "^8.0.0-rc.2",
-    "@storybook/preview": "^8.0.0-rc.2",
-    "@storybook/preview-api": "^8.0.0-rc.2",
```

Comment:
Related to this.
https://github.com/storybookjs/storybook/blob/next/MIGRATION.md#core-changes-and-removals